sdk/metricsadvisor/ai-metrics-advisor/samples/typescript/src/alertingConfig.ts

```diff
@@ -51,7 +50,7 @@ async function createAlertConfig(
   detectionConfigId: string
 ) {
   console.log("Creating a new alerting configuration...");
-  const alertConfig = {
+  const alertConfig: Omit<AnomalyAlertConfiguration, "id"> = {
```

Comment:
why do we need this? @jeremymeng

Reply:
Without the correct type, building samples would give errors like
```
error TS2345: Argument of type '{ name: string; crossMetricsOperator: string; metricAlertConfigurations: ({ detectionConfigurationId: string; alertScope: { scopeType: string; dimensionAnomalyScope?: undefined; }; } | { ...; })[]; hookIds: never[]; description: string; }' is not assignable to parameter of type 'Pick<AnomalyAlertConfiguration, "name" | "description" | "crossMetricsOperator" | "hookIds" | "metricAlertConfigurations">'.
  Types of property 'crossMetricsOperator' are incompatible.
    Type 'string' is not assignable to type '"AND" | "OR" | "XOR" | undefined'.
```